### DISCUSSION TITLE
Numpad Enter dosen't jump back

### DISCUSSION THREAD

**Discussion by JoergM-D** [nguyenquyhy/Flight-Tracker-StreamDeck#367] [Q&A]
Dear all, 
whenever I change the frequency for COM or Nav by uisng the Numpad and hit ENTER, it jumps back to my default profile and not to the prevoius.
How can I change it?

**Comment by nguyenquyhy** [comment]
Sounds like a bug of the Stream Deck software. Have you updated it to version 6.0 yet?

**Comment by JoergM-D** [comment]
Yes, I'm on the latest version

**Comment by JoergM-D** [comment]
I did some tests: If COM, NAV and ADF are in a folder, the numpad jumps back into the profile, not the folder.
